test/test_retryable_writes.py

```diff
@@ -453,6 +454,27 @@ def test_batch_splitting_retry_fails(self):
             self.assertEqual(final_txn, expected_txn)
         self.assertEqual(coll.find_one(projection={'_id': True}), {'_id': 1})
 
+    @client_context.require_version_min(4, 4)
```

Comment:
This also needs `@require_failCommand_fail_point`

Reply:
Done.

Reply:
Why require 4.4? Shouldn't this test also work the same on 4.2?

Reply:
This section https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst#determining-retryable-errors seems to suggest that only MongoDB 4.4+ will add the `RetryableWritesError` to its command response. So the command assertion wouldn't work with MongoDB 4.2

Reply:
That's correct but if I understand correctly, the spec says that on <4.4 the driver adds the RetryableWritesError label to retryable errors. So shouldn't the label be there either way?

Reply:
Please update this to `client_context.require_version_min(3, 6)` to make sure we add the error label correctly on <4.4. We can then make the final assertion server version specific:
```
if version >= 4.4:
        self.assertIn(
            'RetryableWriteError',
            listener.results['succeeded'][-1].reply['errorLabels'])
```